src/FSharp.Analyzers.Cli/Program.fs

```diff
@@ -131,16 +131,23 @@ let printMessages failOnWarnings (msgs: Message list) =
 
     msgs
 
-let calculateExitCode failOnWarnings (msgs: Message list option) : int =
+let calculateExitCode failOnWarnings (msgs: Message list option array) : int =
```

Comment:
Can we not flatten this collection earlier? This type of definition is getting out of hand.

Reply:
Absolutely, good call